package.json

```diff
@@ -57,7 +60,7 @@
   },
   "style": "dist/css/bootstrap.css",
   "sass": "scss/bootstrap.scss",
-  "main": "dist/js/bootstrap",
+  "main": "src/js/index",
```

Comment:
Lazy load is overkill, better use named experts for each plugin/module so that people can import just what they need